app/mailers/foreman_openscap/policy_mailer.rb

```diff
@@ -6,7 +6,7 @@ def policy_summary(options = {})
       user = ::User.find(options[:user])
       @time = options[:time] || 1.day.ago
 
-      @policies = Scaptimony::Policy.all.reject {|policy| policy.assets.map(&:host).compact.empty?}
+      @policies = Policy.all.reject {|policy| policy.assets.map(&:host).compact.empty?}
```

Comment:
@shlomizadok: This works because we are inside ForemanOpenscap module. I will change it to be more explicit, I've just find out I need to update data in taxable_taxonomies table, so I will be making a new commit anyway.